cypress/e2e.sh

```diff
@@ -19,7 +19,7 @@ OPTIONS:
 EOF
 }
 
-SPEC_PATH="cypress/integration/**"
+SPEC_PATH="cypress/e2e/0*"
```

Comment:
Why does the regular expression contains a `0`? Would this be an issue if the app has 10 specs or more?

Reply:
Ooo good point! I will correct this! Thank you Johanna!